### PR TITLE
Focus states: Content Switcher

### PR DESCRIPTION
**Pull request by marijohannessen**
## Overview

Resolves https://github.ibm.com/Bluemix/design-system-website/issues/1054

Updated the focus states for Content Switcher.

### CODE REVIEW

**Comment by tw15egan** [comment]
👍 ✅

File: src/components/content-switcher/_content-switcher.scss [hunk 1]
```diff
@@ -26,15 +26,18 @@
...
     }
 
     &:hover {
       cursor: pointer;
     }
 
     &:hover,
-    &:active {
+    &:active,
+    &:focus {
```
Comment (hellobrian) [inline src/components/content-switcher/_content-switcher.scss]: You can more closely match what the designers intended by doing this instead:

```scss
&:focus {
  border-color: $brand-02;
  text-decoration: underline;
}

&:hover,
  &:active {
    background-color: $brand-02;
    color: $inverse-01;
}
```

Sketch:

image

Comment (marijohannessen) [reply src/components/content-switcher/_content-switcher.scss]: Ah I must have missed that! Fixed it now @hellobrian

**Comment by hellobrian** [comment]
👍 ✅

**Pull request merged**